composer.json

```diff
@@ -17,7 +17,8 @@
     "require": {
         "php": "^5.3.9|^7.0",
         "symfony/framework-bundle": "~2.3|~3.0",
-        "white-october/pagerfanta-bundle": "~1.0"
+        "white-october/pagerfanta-bundle": "~1.0",
+        "twig/twig": "~1.12"
```

Comment:
is this a hard dependency or an optional one? if its optional, we should just add a conflicts section instead.